app/src/main/java/io/apicurio/registry/storage/ArtifactMetaDataDto.java

```diff
@@ -16,13 +16,21 @@
 
 package io.apicurio.registry.storage;
 
-import java.util.Date;
-
 import io.apicurio.registry.rest.beans.ArtifactType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.util.Date;
 
 /**
  * @author eric.wittmann@gmail.com
  */
+@AllArgsConstructor
+@Builder
+@EqualsAndHashCode
+@ToString
```

Comment:
Again - I think this is fine and useful but we'll need to make sure it's compatible with Quarkus native builds.